test/unit/specs/components/DecisioningEngine/createContextProvider.spec.js

```diff
@@ -40,15 +97,51 @@ describe("DecisioningEngine:createContextProvider", () => {
         count: 1
       }
     };
-
     eventRegistry = {
       toJSON: () => events
     };
-    contextProvider = createContextProvider({ eventRegistry });
+    contextProvider = createContextProvider({ eventRegistry }, window);
```

Comment:
I prefer window to be moved within the object, instead of a second parameter.

```js
contextProvider = createContextProvider({ eventRegistry, window });
```